metadata.mk

```diff
@@ -22,14 +22,11 @@ UBI_VERSION=8.10
 
 # Configuration for Semaphore/Github integration.
 ORGANIZATION = projectcalico
+GIT_REPO_NAME = calico
 
 # Part of the git remote that is common to git and HTTP representations.
 # Used to auto-detect the right remote.
-ifeq ($(ORGANIZATION),tigera)
-GIT_REMOTE=$(ORGANIZATION)/calico-private
-else
-GIT_REMOTE=$(ORGANIZATION)/calico
-endif
+GIT_REPO_SLUG ?= $(ORGANIZATION)/$(GIT_REPO_NAME)
```

Comment:
```suggestion
ifeq ($(ORGANIZATION),tigera)
GIT_REPO_NAME=calico-private
endif
GIT_REPO_SLUG ?= $(ORGANIZATION)/$(GIT_REPO_NAME)
```

Reply:
I was thinking it might just be simpler to have metadata.mk by different in calico-private; seems to be the pattern we have elsewhere and I need to cherry-pick this anyway.

Reply:
Yep, this was the intended use of metadata.mk (i.e., it shouldn't have any fancy logic in it, just variables to update)